test/e2e/keygen_test.go

```diff
@@ -0,0 +1,129 @@
+package keygen_test
+
+// import (
+// 	"bytes"
```

Comment:
why the tests are commented out? if it is not working can we fix it? otherwise remove it